catroid/src/main/java/org/catrobat/catroid/common/SharedPreferenceKeys.java

```diff
@@ -29,7 +29,8 @@ private SharedPreferenceKeys() {
 	}
 
 	public static final String ACCESSIBILITY_PROFILE_PREFERENCE_KEY = "AccessibilityProfile";
-	public static final String AGREED_TO_PRIVACY_POLICY_PREFERENCE_KEY = "AgreedToPrivacyPolicy";
+	public static final String AGREED_TO_CURRENT_PRIVACY_POLICY_PREFERENCE_KEY =
```

Comment:
why not just use "int"?
that would spare us all the conversions everytime we use hashCode.
then naming would be AGREED_TO_PRIVACY_POLICY_WITH_THIS_HASH or AGREED_TO_PRIVACY_POLICY_VERSION